azure-communication-ui/chat/README.md

```diff
@@ -0,0 +1,95 @@
+![Hero Image](../../docs/media/mobile-ui-library-chat-hero-image.png)
+
+# Azure Communication UI Mobile Library for Android
+
+Get started with Azure Communication Chat Services by using the UI Library to integrate communication experiences into your applications. For instructions to quickly integrate the UI Library functionalities, please follow our guide below.
+
+
+### Install the packages
+
+In your app level (**app folder**) `build.gradle`, add the following lines to the dependencies and android sections.
+
+```groovy
+android {
+    ...
+    packagingOptions {
+        pickFirst  'META-INF/*'
+    }
+    ...
+}
+```
+
+```groovy
+dependencies {
+    ...
+    implementation 'com.azure.android:azure-communication-ui-chat:0.1.0'
+    ...
+}
+```
+
+In your project gradle scripts add following lines to `repositories`. For `Android Studio (2020.*)` the `repositories` are in `settings.gradle` `dependencyResolutionManagement(Gradle version 6.8 or greater)`. If you are using old versions of `Android Studio (4.*)` then the `repositories` will be in project level `build.gradle` `allprojects{}`.
+
+```groovy
+repositories {
+    ...
+    mavenCentral()
+    maven {
+        url "https://pkgs.dev.azure.com/MicrosoftDeviceSDK/DuoSDK-Public/_packaging/Duo-SDK-Feed/maven/v1"
+    }
+    ...
+}
+```
+Sync project with gradle files. (Android Studio -> File -> Sync Project With Gradle Files)
+
+
+### Quick Sample 
+
+Create `ChatComposite` and launch it. Replace `<USER_ACCESS_TOKEN>` with your token and `<USER_ID>` with your identity string from your azure communication resource's Identity & User Access Tokens settings. Also replace `<DISPLAY_NAME>` with your name and `<THREAD_ID>` with your chat thread ID and `<ENDPOINT_URL>` with your endpoint from your communication resource.
+
+#### [Kotlin](#tab/kotlin)
+
```

Comment:
These code samples will need to be updated.

Reply:
Updated